app/views/providers/proceeding_loop/client_involvement_type/show.html.erb

```diff
@@ -7,12 +7,22 @@
   <%= page_template page_title: @proceeding.meaning, template: :basic, form: do %>
     <span class="govuk-caption-xl"><%= position_in_array(@proceeding) %></span>
     <h1 class="govuk-heading-xl"><%= page_title %></h1>
-    <%= form.govuk_collection_radio_buttons(:client_involvement_type_ccms_code,
-                                            @form.client_involvement_types,
-                                            :ccms_code,
-                                            :description,
-                                            legend: { size: "m", tag: "h2", text: t(".question") }) %>
 
-    <%= next_action_buttons(form:, show_draft: true) %>
+    <% if @legal_aid_application.has_sca_secure_accommodation_order_proceeding? %>
+      <%= form.govuk_radio_buttons_fieldset :client_involvement_type_ccms_code,
+                                            legend: { size: "m", tag: "h2", text: t(".sao_question") } do %>
+        <p class="govuk-body"><%= t(".sao_text") %></p>
+        <%= form.hidden_field :client_involvement_type_ccms_code, value: "W" %>
```

Comment:
I decided against as we only query LFA to get the applicable codes for the current proceeding.  As, in this case, we don't care what the appropriate codes are, it seemed easier to hard code.

For example, we could query LFA that the current proceeding is PB003 and it will say your options are  "D", "W" and "Z", the page will then say as a different proceeding is PB006, you can only see "W"